lib/marcel/mime_type/definitions.rb

```diff
@@ -28,30 +28,30 @@
 Marcel::MimeType.extend "application/vnd.ms-powerpoint.template.macroenabled.12", parents: "application/vnd.openxmlformats-officedocument.presentationml.presentation"
 Marcel::MimeType.extend "application/vnd.ms-powerpoint.slideshow.macroenabled.12", parents: "application/vnd.openxmlformats-officedocument.presentationml.presentation"
 
-Marcel::MimeType.extend "application/vnd.apple.pages", extensions: %w( pages ), parents: "application/zip"
-Marcel::MimeType.extend "application/vnd.apple.numbers", extensions: %w( numbers ), parents: "application/zip"
-Marcel::MimeType.extend "application/vnd.apple.keynote", extensions: %w( key ), parents: "application/zip"
+Marcel::MimeType.extend "application/vnd.apple.pages", parents: "application/zip"
+Marcel::MimeType.extend "application/vnd.apple.numbers", parents: "application/zip"
+Marcel::MimeType.extend "application/vnd.apple.keynote", parents: "application/zip"
```

Comment:
Including the file extensions suggests they're being extended, but they aren't. Omit them to clarify that we're changing the parent.